include/clang/Frontend/LangStandards.def

```diff
@@ -166,7 +166,7 @@ LANGSTANDARD(cuda, "cuda",
 // C++AMP
 LANGSTANDARD(cxxamp, "c++amp",
              "ECMA C++AMP Standard",
-             LineComment | CPlusPlus | CPlusPlus11 | CPlusPlusAMP | Digraphs)
+             LineComment | CPlusPlus | CPlusPlus11 | CPlusPlus14 | CPlusPlusAMP | Digraphs)
```

Comment:
`CPlusPlus1z` is for C++17 support.

If `CPlusPlus14` isn't there, the checks for C++14 fail.